### ISSUE TITLE
Integration with steamcmd container

### ISSUE THREAD

**Issue by Code-Slave** [LorenDB/download-steam-games#4]
Looking at how feasible this would be

https://github.com/steamcmd/docker

provides containers with steamcmd builds. And integrating this with one of them seems like it could be a really clean setup. Just wondering if you think there are any blocker before i even attempt to mess with it.

**Comment by Code-Slave** [comment]
to add to this somewhat is there a way to add id's via a csv or txt file? I have 1k+ games. Im not adding all that by hand  :)

**Comment by LorenDB** [comment]
I don't think that the steamcmd docker image would work out of the box, but it probably wouldn't take a lot of tweaking to make it work. Right now, the script expects a manually installed SteamCMD ([source](https://github.com/LorenDB/download-steam-games/blob/master/source/config.d#L30)), but you could probably abstract that away to allow running it from a Docker container.

Regarding game import, no, that is not possible right now. If you have a file with all your game IDs, you'll have to create your own script to convert your file to JSON as used by the script (try adding one game by hand and then look at `~/.config/download-steam-games/config.json` to see what you need to provide).

**Comment by Code-Slave** [comment]
my thoughts were to add the download-steam-games binary into the steamcmd container build. Id basicall clone that repo and add anything needed to compile, then compile and work out the configs

**Comment by LorenDB** [comment]
Hm, then I don't see a huge blocker to that. I'm not guaranteeing that it will work with zero code changes, but good luck!
